src/locales/ko/mystery-encounters/dark-deal-dialogue.json

```
@@ -1,24 +1,24 @@
 
 
 {
-  "intro": "A strange man in a tattered coat\nstands in your way...",
-  "speaker": "Shady Guy",
-  "intro_dialogue": "Hey, you!$I've been working on a new device\nto bring out a Pokémon's latent power!$It completely rebinds the Pokémon's atoms\nat a molecular level into a far more powerful form.$Hehe...@d{64} I just need some sac-@d{32}\nErr, test subjects, to prove it works.",
-  "title": "Dark Deal",
-  "description": "The disturbing fellow holds up some Pokéballs.\n\"I'll make it worth your while! You can have these strong Pokéballs as payment, All I need is a Pokémon from your team! Hehe...\"",
-  "query": "What will you do?",
+  "intro": "너덜너덜한 외투를 입은 낯선 남자가\n당신을 막아섭니다....",
+  "speaker": "수상한 사람",
+  "intro_dialogue": "거기, 너!$$난 포켓몬의 잠재된 힘을 끌어낼 수 있는\n새로운 장치를 개발 중이거든!$분자 수준에서 포켓몬의 원자를\n훨씬 더 강력한 형태로 완전히 재결합시키는 거야.$히히...@d{64} 약간의 희생- 아니@d{32}\n음, 증명을 도와줄 테스트 대상을 찾고 있어.`",
+  "title": "암거래",
+  "description": "불청객이 볼을 여러 개 들고 있군요.\n\"내가 보답한다니까! 이 강력한 몬스터볼을 대가로 줄 테니까, 너희 팀의 포켓몬을 한 마리 줘! 히히",
```

```suggestion
  "description": "기분 나쁜 남자는 몬스터볼을 여러 개 들고있다.\n\"내가 보장할게! 네 포켓몬을 한 마리만 줘! 대신 아주 좋은 몬스터볼을 줄 테니까. 히히\"",
```
\"\"로 감싸진 부분 누락을 수정했습니다.

---

```suggestion
  "description": "수상한 남자가 몬스터볼을 여러 개 들고 있습니다.\n\"내가 보장할게! 네 포켓몬을 한 마리만 줘! 대신 아주 좋은 몬스터볼을 줄 테니까. 히히\"",
```
위에서 습니다 체로 바꿨어요